content/docs/user-guide/contributing/blog.md

```diff
@@ -0,0 +1,145 @@
+
+```yml
+---
+title: Hello World
+date: '2015-05-01T22:12:03.284Z'
+description: 'Hello World'
+descriptionLong: |
+  Some long
+  multiline
+  text (supports _basic_ [Markdown](link))
+picture: /uploads/image.jpeg
+pictureComment: Some _Comment_ (supports _basic_ [Markdown](link))
+author: ../authors/author_name.md
+tags:
+  - Open Source
+  - Machine Learning
+  - Data Science
+  - Version Control
+  - AI
+---
+
+```
+
+- `title` - **Required.** Title of the post.
+- `date` - **Required.** Publication date in the `YYYY-MM-DD` format. Will be
+  used to sort posts and in RSS.
+- `description` - **Required.** Short description to show in the feed.
+- `descriptionLong` - Optional long description to show before the image on the
+  post page. If not set, `description` will be used instead. Supports basic
+  Markdown markup.
+- `picture` - Optional cover image.
+- `pictureComment` - Optional cover image comment. Supports basic Markdown
+  markup.
+- `author` - **Required.** Relative path to `.md` file with information about
+  the author. See
+  [Adding authors](/doc/user-guide/contributing/blog#adding-authors) to add a
+  new author.
+- `commentsUrl` - Optional link to the [DVC forum](https://discuss.dvc.org)
+  topic. It will contain comments for the post.
+- `tags` - Optional list of tags.
+
+## Content guidelines
+
+It's recommended to follow all the relevant recommendations from the
+[Doc style guidelines](/doc/user-guide/contributing/docs#doc-style-guidelines-javascript-and-markdown).
+
+### Adding images
+
+> 🙏 Please, be sensible about image size you are about to commit, run some
+> online image optimizer. Huge images pollute Git index, slow down `git clone`,
+> deployment, and other operations with the repository.
+
+The original image should be 2x height and width of the one you want to see on
+the screen to look sharp. To add images, put them into `static/uploads` and
+reference like this:
+
+```md
+![](/uploads/images/2020-02-10/image.png)
+```
+
+To add a subtitle:
+
+```md
+![](/uploads/images/2020-02-10/image.png)_subtitle [with a link](link)_
+```
+
+To set the image size and text wrap:
+
+```md
+![](/uploads/images/2020-02-10/image.png '=500')
+![](/uploads/images/2020-02-10/image.png '=500 Some Title')
+![](/uploads/images/2020-02-10/image.png 'Some Title =500')
+![](/uploads/images/2020-02-10/image.png 'Some Title :wrap-left =500')
+![](/uploads/images/2020-02-10/image.png ':wrap-left =500 Some Title')
+```
+
+### Embedding links
+
+Any URLs
+[supported by `gatsby-remark-embedder`](https://github.com/MichaelDeBoey/gatsby-remark-embedder#supported-services)
+such as Youtube and Twitter share links will be embedded automatically. You just
+need to paste them in the post Markdown, for example:
+
+```md
+... Check out this great video right here:
+
+https://youtu.be/4h6I9_xeYA4
+
+And now lets move onto ...
+```
+
+For any other external link you can embed it with a custom HTML-like markup:
```

Comment:
```suggestion
You can ember a preview of any other website using our special  `<external-link>` XHTML tag:
```

Reply:
changed it to a simple language ... not need to precise and frighten users with XHTML :)

Reply:
👍